environments/icds-cas/inventory/postgresql.csv

```diff
@@ -0,0 +1,54 @@
+pgmain0,100.71.184.19,postgresql,,main_dbs,MUMGCCWCDPRDPM00,"[""/dev/vdb""]","[""/dev/vdb1""]",pgmainstandby0,,"[""standby0"",""standby1"",""spare""]",
+pgmainstandby0,100.71.184.28,pg_standby,,main_dbs,MUMGCCWCDPRDPMS0,"[""/dev/vdb""]","[""/dev/vdb1""]",,pgmain0,,standby0
+pgshard0,100.71.184.27,postgresql,,shard_dbs,MUMGCCWCDPRDPS00,"[""/dev/vdb""]","[""/dev/vdb1""]",pgshard0standby,,"[""standby0"",""spare""]",
+pgshard1,100.71.184.29,postgresql,,shard_dbs,MUMGCCWCDPRDPS01,"[""/dev/vdb""]","[""/dev/vdb1""]",pgshard1standby,,"[""standby0"",""spare""]",
+pgshard2,100.71.184.21,postgresql,,shard_dbs,MUMGCCWCDPRDPS02,"[""/dev/vdb""]","[""/dev/vdb1""]",pgshard2standby,,"[""standby0"",""spare""]",
+pgshard3,100.71.184.22,postgresql,,shard_dbs,MUMGCCWCDPRDPS03,"[""/dev/vdb""]","[""/dev/vdb1""]",pgshard3standby,,"[""standby0"",""spare""]",
+pgshard4,100.71.184.32,postgresql,,shard_dbs,MUMGCCWCDPRDPS04,"[""/dev/vdb""]","[""/dev/vdb1""]",pgshard4standby,,"[""standby0"",""spare""]",
+pgshard5,100.71.184.20,postgresql,,shard_dbs,MUMGCCWCDPRDPS05,"[""/dev/vdb""]","[""/dev/vdb1""]",pgshard5standby,,"[""standby0"",""spare""]",
+pgshard6,100.71.184.30,postgresql,,shard_dbs,MUMGCCWCDPRDPS06,"[""/dev/vdb""]","[""/dev/vdb1""]",pgshard6standby,,"[""standby0"",""spare""]",
+pgshard7,100.71.184.31,postgresql,,shard_dbs,MUMGCCWCDPRDPS07,"[""/dev/vdb""]","[""/dev/vdb1""]",pgshard7standby,,"[""standby0"",""spare""]",
+pgshard8,100.71.184.34,postgresql,,shard_dbs,MUMGCCWCDPRDPS08,"[""/dev/vdb""]","[""/dev/vdb1""]",pgshard8tandby,,"[""standby0"",""spare""]",
+pgshard9,100.71.184.26,postgresql,,shard_dbs,MUMGCCWCDPRDPS09,"[""/dev/vdb""]","[""/dev/vdb1""]",pgshard9standby,,"[""standby0"",""spare""]",
+pgshard0standby,100.71.184.47,pg_standby,,shard_standby_dbs,MUMGCCWCDPRDSS00,"[""/dev/vdb""]","[""/dev/vdb1""]",,pgshard0,,standby0
+pgshard1standby,100.71.184.51,pg_standby,,shard_standby_dbs,MUMGCCWCDPRDSS01,"[""/dev/vdb""]","[""/dev/vdb1""]",,pgshard1,,standby0
+pgshard2standby,100.71.184.57,pg_standby,,shard_standby_dbs,MUMGCCWCDPRDSS02,"[""/dev/vdb""]","[""/dev/vdb1""]",,pgshard2,,standby0
+pgshard3standby,100.71.184.49,pg_standby,,shard_standby_dbs,MUMGCCWCDPRDSS03,"[""/dev/vdb""]","[""/dev/vdb1""]",,pgshard3,,standby0
+pgshard4standby,100.71.184.60,pg_standby,,shard_standby_dbs,MUMGCCWCDPRDSS04,"[""/dev/vdb""]","[""/dev/vdb1""]",,pgshard4,,standby0
+pgshard5standby,100.71.184.52,pg_standby,,shard_standby_dbs,MUMGCCWCDPRDSS05,"[""/dev/vdb""]","[""/dev/vdb1""]",,pgshard5,,standby0
+pgshard6standby,100.71.184.58,pg_standby,,shard_standby_dbs,MUMGCCWCDPRDSS06,"[""/dev/vdb""]","[""/dev/vdb1""]",,pgshard6,,standby0
+pgshard7standby,100.71.184.61,pg_standby,,shard_standby_dbs,MUMGCCWCDPRDSS07,"[""/dev/vdb""]","[""/dev/vdb1""]",,pgshard7,,standby0
+pgshard8tandby,100.71.184.56,pg_standby,,shard_standby_dbs,MUMGCCWCDPRDSS08,"[""/dev/vdb""]","[""/dev/vdb1""]",,pgshard8,,standby0
```

Comment:
This is missing an s in standby. left it alone for now, but we may want to change it